src/js/vic-v2/components/PhotoField.jsx

```diff
@@ -439,16 +510,19 @@ export default class PhotoField extends React.Component {
           {!isDone && this.state.src && <div className="cropper-container-outer">
             <Cropper
               ref="cropper"
+              key={smallScreen ? 'smallScreen' : 'largeScreen'}
               ready={this.setCropBox}
               responsive
               src={this.state.src}
               aspectRatio={1}
               cropBoxMovable={false}
               cropend={this.handleCropend}
+              cropmove={this.handleCropMove}
+              minContainerHeight={smallScreen ? 240 : 300}
```

Comment:
For wide images, the container will use the `canvasHeight` that displays the entire image at once as its height. Once this minimum height is set, it cannot be updated without unmounting the component. 

There's an attempt to override the height of the container with CSS which results in the behavior below i.e.
- the cropbox is the calculated container height
- the container is displayed with the overridden height

The cropbox cannot be set to a greater height than the calculated container height.

Forcing the component to 
- meet the minimum 240 / 300 height values in the styling, and 
- swapping the component `key` when the layout changes 

resolves the following issues:

<img width="560" alt="screen shot 2018-02-06 at 20 53 25" src="https://user-images.githubusercontent.com/4998130/35896059-fa4ceedc-0b7f-11e8-829d-a3eacd49e854.png">
<img width="634" alt="screen shot 2018-02-06 at 19 58 07" src="https://user-images.githubusercontent.com/4998130/35896233-eace729a-0b80-11e8-9f98-8ef7963b1b4b.png">